setup.py

```diff
@@ -333,7 +333,12 @@ def has_option(name):
     )]
 
 if cythonize is not None:
-    ext_modules = cythonize(ext_modules)
+    ext_modules = cythonize(ext_modules, compiler_directives={
+        "always_allow_keywords": True,
+        "cdivision": True,
+        "boundscheck": False,
+        "wraparound": False
+    })
```

Comment:
This seems unrelated to this PR and is worth a discussion.

Reply:
It just speed up generated code without extra check.